lib/tasks/webpacker/verify.rake

```diff
@@ -1,17 +0,0 @@
-require "webpacker/configuration"
-
-namespace :webpacker do
-  namespace :install do
-    desc "Verifies if webpacker is installed"
-    task :verify do
-      begin
-        File.read(Webpacker::Configuration.file_path)
```

Comment:
`rails webpacker:verify_install` will show nothing if webpacker is installed.  

But it's not critical.